### ISSUE TITLE
frequent dumps to file

### ISSUE THREAD

**Issue by ryants** [LoLab-VU/PyDREAM#4]
I'd like to do relatively frequent dumps of the chain and log-likelihood to file such that the crossover burnin continues past multiple dumps.  Would it work to set ``crossover_burnin=1`` for multiple ``run_dream`` commands, followed by ``crossover_burnin=0`` for subsequent ``run_dream`` commands once the appropriate number of iterations has been completed?

Let me know if this isn't clear, and I'll provide a code sample.

Also, if you'd like me to direct general questions somewhere other than this repository, let me know

Thanks!

**Comment by eshockley** [comment]
This question and the adapt_gamma problem you fixed highlighted that the way adapt_gamma/adapt_crossover and the gamma and crossover files was being handled was confusing, so I've reworked it. 

Before, if you passed `adapt_crossover = True` and a `crossover_file` to load (or specified a restart run, which loads those files behind the scenes), PyDREAM printed a warning that you had loaded values from a file and asked for adaptation and overrode the adaptation flag, on the assumption that someone would only be adapting those values during the early burn-in period (it violates detailed balance so you don't want to adapt over the whole walk). By default, that adaptation happened over the first 10% of `niter`, with the option to change that value with the `crossover_burnin` parameter. So the value of adapt_crossover was irrelevant on a restart run (or with a file loaded), and it was impossible to do what I think you're asking, which is adapt the values, dump to file, and then continue adapting after restarting.

I can see the argument for wanting to dump to file and continue adapting, so with the latest push I've changed this so that PyDREAM still prints a warning if a file is loaded and adaptation is requested, but proceeds with the adaptation. So to adapt crossover values for the entire first run and then dump to file, then continue adapting for the first 500 iterations of the second run:

```
params, logps = run_dream(parameter_vector, likelihood, niterations=1000, crossover_burnin=1000, adapt_crossover=True, model_name='test_')
params2, logps2 = run_dream(parameter_vector, likelihood, niterations=1000, crossover_burnin=500, adapt_crossover=True, model_name='test_', restart=True)
```
Let me know if this isn't what you meant or is unclear.
